sktime/forecasting/ensemble_algorithms.py

```diff
@@ -0,0 +1,234 @@
+from sklearn.base import BaseEstimator
+from scipy.optimize import nnls
+import numpy as np
+
+
+class EnsembleAlgorithm(BaseEstimator):
```

Comment:
Can we call this `PredictionWeightingEnsembler` or similar? Not all ensembles are prediction weighting ensembles, but yours are.

Reply:
This would make it much clearer what the algorithm actually does!

Reply:
Definitely can change this!